release-notes/unreleased/2250.other.rst

```diff
@@ -0,0 +1 @@
+`IBMInputValueError` inherits from `ValueError`, in addition to the existing parent class `IBMError`. Some validation functions, which previously raised `ValueError` exceptions, now raise `IBMInputValueError` exceptions. We added a validation that observables consist only of Pauli operations, residing in the alphabet {I, X, Y, Z}.
```

Comment:
Here are some minor formatting suggestions. I would add a feature reno for the new validation too!

```suggestion
:class:`.IBMInputValueError` now inherits from ``ValueError``, in addition to the existing parent class :class:`.IBMError`. Some validation functions, which previously raised ``ValueError`` exceptions, now raise :class:`.IBMInputValueError` exceptions. We added a validation that observables consist only of Pauli operations, residing in the alphabet ``{I, X, Y, Z}``.
```

Reply:
Thanks @ElePT for the review. Do you suggest to split to `2250.other.rst` and `2250.feat.rst`, where the last sentence ("We added a validation...") foes to the `feat` file?

Reply:
I suggest splitting it, yes!